composer/datasets/cifar_hparams.py

```diff
@@ -183,25 +184,60 @@ class StreamingCIFAR10Hparams(DatasetHparams):
     """Streaming CIFAR10 hyperparameters.
 
     Args:
+        version (int): Which version of streaming to use.
         remote (str): Remote directory (S3 or local filesystem) where dataset is stored.
             Default: ``'s3://mosaicml-internal-dataset-cifar10/mds/1/'``
         local (str): Local filesystem directory where dataset is cached during operation.
             Default: ``'/tmp/mds-cache/mds-cifar10/'``
         split (str): The dataset split to use, either 'train' or 'val'. Default: ``'train'``.
     """
 
+    version: int = hp.optional('Version of streaming (1 or 2)', default=1)
     remote: str = hp.optional('Remote directory (S3 or local filesystem) where dataset is stored',
                               default='s3://mosaicml-internal-dataset-cifar10/mds/1/')
     local: str = hp.optional('Local filesystem directory where dataset is cached during operation',
                              default='/tmp/mds-cache/mds-cifar10/')
     split: str = hp.optional("Which split of the dataset to use. Either ['train', 'val']", default='train')
 
     def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHparams) -> DataLoader:
-        dataset = StreamingCIFAR10(remote=self.remote,
-                                   local=self.local,
-                                   split=self.split,
-                                   shuffle=self.shuffle,
-                                   batch_size=batch_size)
+        if self.version == 1:
+            dataset = StreamingCIFAR10(remote=self.remote,
+                                       local=self.local,
+                                       split=self.split,
+                                       shuffle=self.shuffle,
+                                       batch_size=batch_size)
+        elif self.version == 2:
+            try:
+                from streaming.vision import CIFAR10
+            except ImportError as e:
+                raise MissingConditionalImportError(extra_deps_group='streaming',
+                                                    conda_package='mosaicml-streaming') from e
+
+            cifar10_mean = 0.4914, 0.4822, 0.4465
```

Comment:
should we make it UPPER_CASE since it is a constant and doesn't required any data change? E.g.  `CIFAR10_MEAN = 0.4914, 0.4822, 0.4465` ?

Reply:
Done